python/pyspark/streaming/dstream.py

```diff
@@ -602,13 +602,17 @@ class TransformedDStream(DStream):
     Multiple continuous transformations of DStream can be combined into
     one transformation.
     """
-    def __init__(self, prev, func):
+    def __init__(self, prev, func, transformFunc=None):
```

Comment:
Actually, I misunderstood this. And I am not really getting why this is necessary.